mehmetsaidozsoy.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Mehmet Said ÖZSOY",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " Motorcycle lover",
+  "avatar": "https://avatars.githubusercontent.com/u/6255955?s=40&v=4",
```

```suggestion
  "avatar": "https://github.com/mehmetsaidozsoy.png",
```
Please use this icon